source/includes/limitations/limits.rst

```diff
@@ -80,40 +75,58 @@ in the MongoDB manual.
 Mutual Recursion
 ~~~~~~~~~~~~~~~~
 
-.. important:: Impacted Versions
-
-   - 4.3
-   - 4.4
-
-You cannot specify a **mutually recursive** type as a type parameter.
+In versions 4.3 through 4.10 of the {+driver-short+}, you cannot specify a 
+**mutually recursive** type as a type parameter. To specify a mutually 
+recursive type as a type parameter, use version 4.11 or newer.
 
 A mutually recursive type exists when two types contain a property that is of
-the other's type. You can update the
-:ref:`Pet <mongodb-node-typescript-pet-interface>` interface
-to be mutually recursive by allowing a pet to have a handler, and defining a
-handler to have a pet. The following are the mutually
+the other's type. You can update the :ref:`Pet <mongodb-node-typescript-pet-interface>` 
+interface to be mutually recursive by allowing a pet to have a handler, and 
+defining a handler to have a pet. The following examples reference the mutually
 recursive ``Pet`` and ``Handler`` interfaces:
 
 .. code-block:: typescript
    :emphasize-lines: 2, 8
 
-   interface MutuallyRecursivePet {
+   interface Pet {
       handler?: Handler;
       name: string;
       age: number;
    }
 
    interface Handler {
-      pet: MutuallyRecursivePet;
+      pet: Pet;
       name: string;
    }
+   
+The {+driver-short+} provides type safety for mutually recursive types 
+referenced through dot notation up to a depth of eight. The following code 
```

Comment:
[v4.11](https://github.com/mongodb/node-mongodb-native/releases/tag/v4.11.0#:~:text=dot%20notation%20queries-,up%20to%20a%20depth%20of%209,-.%20Beyond%20a%20depth) release notes say nine, but tests of original PR say [eight](https://github.com/mongodb/node-mongodb-native/pull/3433/files#diff-dbf13828200bb3db6c3c43ff643f181c69855415455caaf7a48b1b3a88eae954R72:~:text=%3E(%7B-,%27bestBook.author.bestBook.author.bestBook.author.bestBook.author.name%27%3A%2023,-%7D)%3B)

Reply:
The tests are correct, I'll be updating the notes